### PR TITLE
Feature/414 API migration - Part 1

### PR DESCRIPTION
**Pull request by dweemx**
Part 1 of #414 
Complete switch (`gRPC` implementation is removed) of the following endpoints to `REST` `FastAPI`:
- `getMyLooms`
- `getRemainingUUIDTime`
- `getUUID`

### CODE REVIEW

File: opt/main.py [hunk 1]
```diff
@@ -3,33 +3,71 @@
...
+from scopeapi.model import GetRemainingUUIDTime, GetDatasets
 
 scope_api = FastAPI()
+scope_api.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
```
Comment (MaybeJustJames) [inline opt/main.py:25]: This is tricky. Maybe ok for now.

File: opt/main.py [hunk 2]
```diff
@@ -3,33 +3,71 @@
...
+
 CONFIG = from_file(os.environ.get("SCOPE_CONFIG"))
 
+__scope_server = SCopeServer(config=CONFIG)
+__scope_server_legacy = LegacySCopeServer(config=CONFIG, new_server=__scope_server)
+__scope_api = SCopeAPI(server=__scope_server)
```
Comment (MaybeJustJames) [inline opt/main.py:31]: Is there any reason you're using a dunder prefix here? I would prefer you avoid it.

File: opt/main.py [hunk 3]
```diff
@@ -3,33 +3,71 @@
...
+
 
-@scope_api.get("/echo/{value}")
+@scope_api.get("/echo/{value}/")
```
Comment (MaybeJustJames) [inline opt/main.py:34]: This endpoint can be removed.

**Review by MaybeJustJames — CHANGES_REQUESTED** [review]
A general comment on organisation:

The top-level seperation of `scopeapi` and `scopeserver` is a mistake because there are guaranteed to be dependencies between them. Here's the organisation I would prefer to see:

```
.
├── alembic/
│   └── ...
├── alembic.ini
├── main.py
├── scopeserver
│   ├── api
│   │   ├── deps.py
│   │   ├── __init__.py
│   │   └── v1
│   │       ├── __init__.py
│   │       └── ...
│   ├── config.py
│   ├── database.py
│   ├── __init__.py
│   ├── models.py
│   └── schemas.py
└── tests
      └── ...
```

File: opt/main.py (hunk 1 above)
Comment (dweemx) [reply opt/main.py:25]: Yes did not find a better solution yet.

File: opt/main.py (hunk 2 above)
Comment (dweemx) [reply opt/main.py:31]: I will change that

**Pull request closed without merging**